.gitignore

```diff
@@ -1,7 +1,2 @@
-_site/
-.sass-cache/
-.jekyll-cache/
-.jekyll-metadata
-.DS_Store
-docs/.DS_Store
-.vscode
+.gitignore
+.ruby-version
```

Comment:
Did you mean to remove the other lines in this file?

Reply:
Oh heck. I didn't mean to commit this file at all! Thanks for catching that.

Reply:
Should be sorted out now!